homeassistant/components/frontier_silicon/media_player.py

```diff
@@ -42,6 +43,8 @@
     }
 )
 
+SCAN_INTERVAL = timedelta(seconds=15)
```

Comment:
This looks like a feature change. The default scan interval is 10 seconds for media players. Why do we change it? If we need to change it, please move it to a separate PR.

Reply:
This was the result of trying to follow the rules defined in [Integration Quality Scale](https://developers.home-assistant.io/docs/integration_quality_scale_index): 

> Set an appropriate SCAN_INTERVAL (if a polling integration)

I've removed it again, as the default is indeed ok.